src/relayclient/GSNConfigurator.ts

```diff
@@ -122,6 +122,7 @@ export interface GSNConfig {
   relayLookupWindowBlocks: number
   methodSuffix: string
   jsonStringifyRequest: boolean
+  requiredVersionRange?: string
```

Comment:
The problem is that the default is not fixed, but based on current version, in code.
I can put default as "null", but that's the same as ?:

Reply:
I can 'default' it to an empty string, and then the logic of using the current version will be
 based on `if (requiredVersionRange=='')` 
instead of `if (requiredVersionRange==null)`
is it any better?

Reply:
But current version is fixed at compile time, so I don't get your point. Default it to current version but not in some class constructor but right here, what's the difference?

Reply:
we construct VersionManager with current version - I left that as it is, only added it a paraemter of requiredVersion (which can be based on current version, but doesn't have to)

Do you suggest is to remove the VersionManager's version, and use the global version code as default requiredVersion?